web_server/authors/static/css/posting.css

```diff
@@ -0,0 +1,61 @@
+@charset "UTF-8";
+
+body {
```

Comment:
For consistency, CSS on bare tags should be extracted to a global site CSS file that gets loaded in the index.html -> otherwise basic elements may appear differently on different pages.